coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Will need to remove coverage- it was modified in this branch after merge of #30, so it will make it back in